3001-4000/LIT3326Melchiz1.xml

```diff
@@ -56,11 +76,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="MV" when="2017-03-20">Added bibliography and keywords</change>
          <change who="MV" when="2017-12-27">Added clavis ID and keywords</change>
          <change who="DR" when="2018-11-27">Specified clavis</change>
+         <change who="NV" when="2024-11-11">Added an abstract, a list of witnesses, incipit and explicit,  more bibl</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="bibliography">
+            <listRelation>
+               <relation name="ecrm:P129_is_about" active="LIT2365Storyo" passive="PRS6988Melchize"/>
```

Comment:
there is already a list of relations below, you can add this relation to the existing list